data/json/items/gun/ups.json

```diff
@@ -116,7 +116,7 @@
     "id": "v29",
     "type": "GUN",
     "name": "V29 laser pistol",
-    "description": "The V29 laser pistol was designed in the mid-21st century and was one of the first firearms to use fusion as its ammunition.  It is larger than most traditional handguns, but displays no recoil whatsoever.",
+    "description": "The V29 laser pistol was designed in the mid-21st century and was one of the first handheld laser weapons.  It is larger than most traditional handguns, but displays no recoil whatsoever.",
```

Comment:
This "mid-21st century" should be changed, as we now have agreement that the game take action in the early 21st century.

Reply:
Might move the description to BrightNights or Icecoon's Arsenal